src/EPR.SubsidiaryBulkUpload.Application.UnitTests/Services/RecordExtractionTests.cs

```diff
@@ -56,10 +56,13 @@ public void ShouldIgnoreRecordsWhereThereAreNoParents()
         var extraction = new RecordExtraction();
 
         // Act
-        var parentAndSubsidiaries = extraction.ExtractParentsAndSubsidiaries(subsidiaries);
+        var parentAndSubsidiaries = extraction.ExtractParentsAndSubsidiaries(subsidiaries).ToArray();
 
         // Assert
-        parentAndSubsidiaries.Should().BeEmpty();
+        parentAndSubsidiaries.Should().HaveCount(3);
+        parentAndSubsidiaries[0].Parent.organisation_name.Should().BeEquivalentTo("orphan");
+        parentAndSubsidiaries[1].Parent.organisation_name.Should().BeEquivalentTo("orphan");
+        parentAndSubsidiaries[2].Parent.organisation_name.Should().BeEquivalentTo("orphan");
```

Comment:
You could use `.Should().AllBeEquivalentTo("orphan")` here and below

Reply:
all three parents are not same. Id are different. only the name is used like that.